content/en/blog/_posts/2020-11-02-remembering-dan-kohn.md

```diff
@@ -4,10 +4,10 @@ title: "Remembering Dan Kohn"
 date: 2020-11-02
 slug: remembering-dan-kohn
 evergreen: true
+author: >
+  The Kubernetes Steering Committee
```

Comment:
```suggestion
  Kubernetes Steering Committee
```
will render as
> by **Kubernetes Steering Committee**

“the” is OK, but will look odd if bolded

Reply:
Addressed in latest commit.